app/src/main/java/org/eyeseetea/malariacare/domain/entity/PushReport.java

```diff
@@ -0,0 +1,85 @@
+package org.eyeseetea.malariacare.domain.entity;
```

Comment:
@idelcano I think it would be convenient to group the concepts related to pushreport within the same package, for example Push, PushResponse or PushSummary or another similar name